api/modules/city/views.py

```diff
@@ -141,3 +147,23 @@ def get_city_trends(request, city_id):
         return Response(str(e), status=status.HTTP_503_SERVICE_UNAVAILABLE)
 
     return Response(response)
+
+
+@api_view(['GET'])
+def get_city_visits(request):
+    """
+        Returns a list of cities visited by a user
```

Comment:
format your comments correctly

Reply:
@prabhakar267  sorry but i am not getting what do you mean by logs